openverse_catalog/dags/common/constants.py

```diff
@@ -7,6 +9,8 @@
 IMAGE = "image"
 MEDIA_TYPES = [AUDIO, IMAGE]
 
+MEDIA_TYPES_TYPE = Literal[AUDIO, IMAGE]
```

Comment:
Oh wait! [It seems like we actually can](https://stackoverflow.com/a/68474515/3277713)! 😮 

Also I should have mentioned this before, and this is a total nit, but - python types are usually declared in CamelCase, since they're not really constants and that matches the definitions in the `typing` library.
```suggestion
MEDIA_TYPES = [AUDIO, IMAGE]

MediaType = Literal[tuple(MEDIA_TYPES)]
```

Reply:
I saw that source, but it goes on to say:
>  However, literal types cannot contain arbitrary expressions: types like Literal[my_string.trim()], Literal[x > 3], or Literal[3j + 4] are all illegal." So this is valid python syntax, but will not be understood by any type checker, which completely defies the point of adding type hints in the first place

Do you think that's still worth it?

Reply:
Actually, seems like it won't allow constants either. So while it allows this:
`MediaType = Literal('audio', 'image')`
It complains about:
`MediaType = Literal( AUDIO, IMAGE )`

Reply:
Aww, that's a shame. Apologies for the confusion on my part there. No, I think it's all fine as-is!

Reply:
Won't the accepted answer on that SO link Madison shared work?

Reply:
Not in a way that feels clean enough to me. We can do this:

```
# Define constants
AUDIO = 'audio'
IMAGE = 'image'

# Define the type; we cannot pass AUDIO and IMAGE in here
MediaType = typing.Literal['image', 'audio']

# Build MEDIA_TYPES constant from the type
MEDIA_TYPES: typing.Tuple[MediaType, ...] = typing.get_args(MediaType)
```

But we still have to add media types in multiple places, and I think it becomes less readable.